hassio/src/dashboard/hassio-update.ts

```diff
@@ -0,0 +1,150 @@
+import {
+  LitElement,
+  TemplateResult,
+  html,
+  CSSResult,
+  css,
+  property,
+  customElement,
+} from "lit-element";
+
+import { HomeAssistant } from "../../../src/types";
+import {
+  HassioHomeAssistantInfo,
+  HassioHassOSInfo,
+  HassioSupervisorInfo,
+} from "../../../src/data/hassio";
+
+import { hassioStyle } from "../resources/hassio-style";
+
+import "@material/mwc-button";
+import "@polymer/paper-card/paper-card";
+import "../../../src/components/buttons/ha-call-api-button";
+import "../components/hassio-card-content";
+
+@customElement("hassio-update")
+export class HassioUpdate extends LitElement {
+  @property() public hass?: HomeAssistant;
```

Comment:
We can start using `hass!` pretty much everywhere, it's always there :) 

```suggestion
  @property() public hass!: HomeAssistant;
```